identifiers/country-fr.csv

```diff
@@ -0,0 +1,699 @@
+id,name
+ocd-division/country:fr,France
+ocd-division/country:fr/oversea_department:francais_etablis_hors_de_france,Français établis hors de France
+ocd-division/country:fr/oversea_department:francais_etablis_hors_de_france/ed:1,Première circonscription des Français établis hors de France
```

Comment:
Didn't we start using the special id `diaspora` in other PRs? I'd prefer that consistency, instead of `francais_etablis_hors_de_france`.

Reply:
sounds reasonable, is implemented